feat(web): Improve duplicate suggestion

Sese-Schneider opened the pull request:
## Description

Improves the algorithm to select the suggested asset on the deduplication page and for mass deduplication.

In addition to the file size, the algorithm now also takes into account the amount of EXIF data present, if multiple assets have the best quality.

In order to make the reasoning of the suggestion more transparent to the user, I also added a info popup to the `DuplicateAsset` view.

### Screenshots

<img src="https://github.com/user-attachments/assets/67657d29-02f8-4e30-8484-f6b704a8fcf0" height="512px">

Sese-Schneider committed:
feat: Improve duplicate suggestion

Sese-Schneider committed:
format

alextran1502 commented:
I wonder if it would be helpful to explain to the user which criteria are taken into account in mass deduplication by placing them in a popup sheet, which is shown when hovering over an info icon next to this line.

<img width="262" alt="image" src="https://github.com/user-attachments/assets/f14ce4a6-fc10-49bc-865e-af5565357fb7" />

Sese-Schneider commented:
@alextran1502 do you suggest I implement that change in this PR?

alextran1502 commented:
I think it will help, from looking at the UI, I don't understand right away what "x EXIF" mean

Sese-Schneider committed:
feat(web): Add deduplication info popup

Sese-Schneider commented:
@alextran1502 I've added the popup as you requested, and also adjusted the algorithm to be more readable.
Are there any additional steps required from my side? Is it okay to just add string and translation via PR?

<img src="https://github.com/user-attachments/assets/f4e57b8a-c316-4437-bfd5-96650df95a5f" height="512px">

Sese-Schneider committed:
Merge branch 'main' into feat/update-duplicate-suggestion

alextran1502 reviewed (approved):
Looks good! Thank you.

alextran1502 committed:
fix: lint

alextran1502 committed:
Merge branch 'main' into feat/update-duplicate-suggestion

Sese-Schneider committed:
fmt

Sese-Schneider committed:
Merge remote-tracking branch 'Sese-Schneider/feat/update-duplicate-suggestion' into feat/update-duplicate-suggestion

alextran1502 commented:
@Sese-Schneider No problem! Thank you
